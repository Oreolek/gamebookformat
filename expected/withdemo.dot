digraph gamebook {

 1->5

1->2

1->4
2->4
4->2

4->3
5->3
}
