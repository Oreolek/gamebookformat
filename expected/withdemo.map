{"demo2": 4, "start": 1, "demo1": 5, "theend": 3, "demoinfo": 2}