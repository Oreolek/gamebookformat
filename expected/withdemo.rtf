{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Gamebook
\b0\
\b Turn to 1 to begin.
\b0\
\
\b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  This gamebook contains a few sections tagged as demo, that will be included when building the example (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. Go on to \b {\field{\*\fldinst  REF s5 \\h }{\fldrslt 5}}
\b0
 or \b {\field{\*\fldinst  REF s4 \\h }{\fldrslt 4}}
\b0
 or \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
.  \
\

\b \qc {\*\bkmkstart s2}2{\*\bkmkend s2}
\b0\
\ql  This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Go on to \b {\field{\*\fldinst  REF s4 \\h }{\fldrslt 4}}
\b0
.  \
\

\b \qc {\*\bkmkstart s3}3{\*\bkmkend s3}
\b0\
\ql  This is the end. It is also included in the demo.  \
\

\b \qc {\*\bkmkstart s4}4{\*\bkmkend s4}
\b0\
\ql  This is the second demo section. You can go on to the end at \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
 or alternative (not in demo) end at \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
.  \
\

\b \qc {\*\bkmkstart s5}5{\*\bkmkend s5}
\b0\
\ql  This is the first demo section (not counting the start section). You can go on to the end at \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
.  \
\

}
