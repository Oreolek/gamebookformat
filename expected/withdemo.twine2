<tw-storydata name="Gamebook" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> This gamebook contains a few sections tagged as demo, that will be included when building the example (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. Go on to [[5]] or [[4]] or [[2]]. </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Go on to [[4]]. </tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> This is the end. It is also included in the demo. </tw-passagedata>
<tw-passagedata pid="4" name="4" tags="" position="40,40"> This is the second demo section. You can go on to the end at [[3]] or alternative (not in demo) end at [[2]]. </tw-passagedata>
<tw-passagedata pid="5" name="5" tags="" position="50,50"> This is the first demo section (not counting the start section). You can go on to the end at [[3]]. </tw-passagedata>
</tw-storydata>
