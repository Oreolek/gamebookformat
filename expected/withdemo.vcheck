examples/withdemo.debug: ending (death?) section found: theend
