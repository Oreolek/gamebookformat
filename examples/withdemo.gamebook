missingto = demoinfo

* 1 start :demo:
This gamebook contains a few sections tagged as demo, that will
be included when building the example
(see withdemo.gamebook.options file). There are also some sections
without the demo tag that will be ignored.
Go on to [[demo1]] or [[demo2]] or [[notindemo1]].

* notindemo1
This section is not included in the demo. Turn to [[notindemo2]].

* demo1 :demo:
This is the first demo section (not counting the start section).
You can go on to the end at [[theend]].

* notindemo2
This section is not included in the demo. You can go on to
[[demo2]] or [[theend]] or [[altend]].

* demo2 :demo:
This is the second demo section.
You can go on to the end at [[theend]] or alternative (not in demo)
end at [[altend]].

* theend :demo:
This is the end. It is also included in the demo.

* demoinfo :demo:
This is where references to non-demo sections will lead you.
It is configured at the top of the gamebook file.
Remember to tag this section correctly to be included.
Go on to [[demo2]].

* altend
Alternative end. Not included in the demo.
