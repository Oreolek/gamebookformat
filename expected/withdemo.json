{
  "title" : "Gamebook",
  "nr_sections" : "5",

  "sections" : {"IGNORE-debug-json-padding-IGNORE" : "", "start" : {
  "nr" : 1,
  "text" : ["" ," This gamebook contains a few sections tagged as demo, that will be included when building the example (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. Go on to ",{"reference" : "5"}," or ",{"reference" : "4"}," or ",{"reference" : "2"},". "]
}
, "demoinfo" : {
  "nr" : 2,
  "text" : ["" ," This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Go on to ",{"reference" : "4"},". "]
}
, "theend" : {
  "nr" : 3,
  "text" : ["" ," This is the end. It is also included in the demo. "]
}
, "demo2" : {
  "nr" : 4,
  "text" : ["" ," This is the second demo section. You can go on to the end at ",{"reference" : "3"}," or alternative (not in demo) end at ",{"reference" : "2"},". "]
}
, "demo1" : {
  "nr" : 5,
  "text" : ["" ," This is the first demo section (not counting the start section). You can go on to the end at ",{"reference" : "3"},". "]
}
}}
